react/react1/week1/src/components/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import Note from "./Note";
+
+function App() {
+  return (
+    <div>
+      <Header />
+      <div className="note">
+        <Note />
+      </div>
+      <Footer />
+    </div>
```

Comment:
I can see that this div is mostly working as a wrapper, in React you can use <></> instead, so that you do not create an extra div element that is not really needed